data/human/deep missions.txt

```diff
@@ -2752,3 +2752,1453 @@ mission "Stone of our Fathers 5"
+			`	She points to a small wooden craft up ahead. "I built her." She says with pride. "No more Ice-Crawler for Hjlod. I am sailor now. I take curious tourist out now and again. Much safer than Ice-Crawler, except when we capsize." She shrugs and grins. "Perhaps someday you'll be my tourist again. This time, I promise no accidents. Also, there are no Ice-Slugs to worry about." She pauses for a moment. "There are Shark-Kraken though. But don't worry. Hjlod will take care of you."`
+			`	You spend the afternoon in Hrithfjall, where you are treated like a guest of honor. The Ondurdis seems younger than you remember, and also appears quite fond of old Aldfrith. You eat strange fish, listen to unlikely stories, and laugh at outrageous jokes.`
+			`	As the sun sinks in the sky, you announce that you should be heading back. "Wait," the Ondurdis says. "I have task for you. If you are interested."`
+			choice
+				`	"What is it?"`
+					goto task
+				`	"Sure; I'd do anything for you guys."`
+					goto task
+				`	"Hard pass."`
+			`	You explain you've already done enough for them and are needed elsewhere. They graciously accept your excuses, and you leave back to your ship.`
+				decline
+			label task
+			`	The Ondurdis smiles. "I knew you would accept.`
+			`	"We have folk here who have died over last year. Mostly old, a few in accidents, nothing out of ordinary. But each Skadenga carries with them life-stone, upon which they carve their memories. When Skadenga dies, these stones must be returned to their Family-Cairns by their nearest relative or closest friend. But, all our Family-Cairns are on <planet>."`
+			`	She then nods at you. "If you could transport family members of deceased to return their stones, it would mean much to everyone here. We would pay you <payment>."`
+			choice
+				`	"Of course."`
+					goto yes
+				`	"I'll do it for free."`
+					goto yesyes
+				`	"No thanks."`
+			`	You explain to them you have more pressing matters to attend to and cannot be a taxi service for their grieving. They graciously accept your excuses, and you leave back to your ship.`
+				decline
+			label yesyes
+			`	The Ondurdis shakes her head. "You have already done too much for us for too little pay. We can afford to pay you, and we will."`
+			label yes
+			`	"The five Stone-Bearers will accompany you back to your ship. May Skade and Ran watch over your journey." The Ondurdis turns this into a toast, which everyone gladly participates in.`
+			`	Hjlod walks with you and your passengers back to your ship. "Be well, <first>."`
+				accept
+	on complete
+		payment 50000
+		event "hjlod waits" 360
+
+
+
+mission "Stones of Skadenga 1"
```

Comment:
Is this mission really necessary? The norm for the game is that "out-and-back" missions pay upfront, and that's what I assumed was the case when I read the preceding mission. I just thought bringing them back to Windblain was part of the deal for the amount they were offering to pay me.
I'd personally just combine this with the one above into a stopover mission and remove the option to refuse to take them back, unless you're dead-set on keeping it. If you are, I'd be curious what the reasoning is.

Reply:
I have combined one of these missions, but the very first one is still split into two parts, however I removed the option to decline it and leave the stone-bearers stranded. The reason I've left it in two parts is because there is a fair bit going on and I am not confident enough in my abilities to encapsulate it all in a single stopover mission. However, with the decline option removed, it functions very similar to a stopover mission now.